libs/portal/features/profile/hooks/use-profile-schema.ts

```
@@ -0,0 +1,125 @@
+export function useProfileSchema(
+  user: Partial<UserDetails>,
+  isStudent: boolean,
+) {
+  const profileSchema = z.object({
+    firstName: z
+      .string()
+      .max(100)
+      .refine((value: string) => value !== null && value !== undefined, {
+        message: 'Required',
+      })
+      .refine((value) => /^[a-z ]+$/i.test(value), {
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)